certified-connectors/Seismic Configuration/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "0718e920-53ef-48e9-994c-504906a6e540",
```

Comment:
@rkushvaha secret look alike values are not allowed. please replace with dummy values.

Reply:
Added dummy value